main.rb

```diff
@@ -0,0 +1,8 @@
+require './app'
+
+def launch
+  app = App.new
+  app.start_app
```

Comment:
- [ ] Kindly I  suggest you use the method ` run ` instead of `start_app` it will remove the bag in your terminal when executing the main.rb  file :+1: